tools/translation_manager.py

```diff
@@ -1,38 +1,43 @@
 #!/usr/bin/env python3
 
+import json
+import copy
+import glob
+
 import fire
-import json, copy, glob
 
-LOCALE_DIR="../locales"
 
-INDENT=4
+LOCALE_DIR = "../locales"
+INDENT = 4
 
 
-def openLocaleFile(fname):
+def open_locale_file(fname):
     """Opens a locale file"""
-    with open(fname) as json_file: 
-      return json.load(json_file) 
+    with open(fname) as json_file:
+        return json.load(json_file)
+
 
-  
 def write_json(data):
```

Comment:
Should we rename this to `write_sorted_json`, or include a note in the doc comments regarding the sorting? I think that's the key feature of this method  (along with our standard indent value) compared to just using `json.dumps()` straight up. This confused me a bit the first time I looked at the `alphabetize` public method.